src/components/Input/CreditCardNumber.tsx

```diff
@@ -33,6 +40,7 @@ const ICONS: {
   'master-card': 'cc-mastercard',
   discover: 'cc-discover',
   jcb: 'cc-jcb',
+  unionpay: 'credit-card',
```

Comment:
There is no Font Awesome icon for UnionPay, so I just used the default credit card icon